inventories/terra-env/group_vars/stackdriver.yml

```diff
@@ -0,0 +1,3 @@
+---
+thurloe_gitrepos__to_merge:
```

Comment:
maybe `stackdriver` insteadd of `thurloe` here.

Reply:
oof yep